lightningd/subd.c

```diff
@@ -510,3 +511,20 @@ char *opt_subd_dev_disconnect(const char *optarg, struct lightningd *ld)
 			       optarg, strerror(errno));
 	return NULL;
 }
+
+/* If test specified that this disconnection should cause permanent failure */
+bool dev_disconnect_permanent(struct lightningd *ld)
+{
+	char permfail[strlen("PERMFAIL")];
```

Comment:
We could #define it, but it's ugly any way we do it.  It's also dev code, so I'm not hugely concerned...